state-chain/custom-rpc/src/lib.rs

```diff
@@ -1730,6 +1742,18 @@ where
 	) -> RpcResult<Vec<u8>> {
 		self.with_runtime_api(at, |api, hash| api.cf_filter_votes(hash, validator, proposed_votes))
 	}
+
+	fn cf_subscribe_tainted_btc_transaction_events(
+		&self,
+		sink: jsonrpsee::PendingSubscriptionSink,
+	) {
+		self.new_subscription(
+			false, /* only_on_changes */
```

Comment:
What does 'only on changes = false' mean? We would get an update per block? I think only pushing an update when there is an event should be enough?

Reply:
I agree, now '=true'.